### PR TITLE
helm-dashboard: init at 1.3.1

### PR DESCRIPTION
**Pull request by qjoly**
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by tomodachi94 — APPROVED** [review]
Lgtm.

**Comment by geri1701** [comment]
Result of `nixpkgs-review pr 231299` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>helm-dashboard</li>
  </ul>
</details>

**Review by geri1701 — APPROVED** [review]

File: pkgs/applications/networking/cluster/helm-dashboard/default.nix [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{ lib, buildGoModule, fetchFromGitHub, installShellFiles }:
```
Comment (ambroisie) [inline pkgs/applications/networking/cluster/helm-dashboard/default.nix:1]: `installShellFiles` is not being used:

```suggestion
{ lib, buildGoModule, fetchFromGitHub }:
```

File: pkgs/applications/networking/cluster/helm-dashboard/default.nix [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  src = fetchFromGitHub {
+    owner = "komodorio";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-D9da40+DbU1EMdR/a4ahLtqlzwPdcHOiAJtPjKZ2Ehc=";
+  };
+
+  vendorHash = "sha256-LJVL20CsDxaAJ/qS+2P7Pv/jhyRO6WAmhGLCR9CmQKE=";
+
+  doCheck = false;
```
Comment (ambroisie) [inline pkgs/applications/networking/cluster/helm-dashboard/default.nix:16]: Add a comment explaining why you had to disable the tests.

**Review by ambroisie — CHANGES_REQUESTED** [review]

**Review by ambroisie — APPROVED** [review]

File: pkgs/applications/networking/cluster/helm-dashboard/default.nix [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  src = fetchFromGitHub {
+    owner = "komodorio";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-D9da40+DbU1EMdR/a4ahLtqlzwPdcHOiAJtPjKZ2Ehc=";
+  };
+
+  vendorHash = "sha256-LJVL20CsDxaAJ/qS+2P7Pv/jhyRO6WAmhGLCR9CmQKE=";
+
+  # # disable tests that require internet access
```
Comment (SuperSandro2000) [inline pkgs/applications/networking/cluster/helm-dashboard/default.nix:16]:
```suggestion
  # disable tests that require internet access
```

**Pull request merged**